debian/control

```diff
@@ -0,0 +1,63 @@
+Source: tsc
+Section: games
+Priority: optional
+Maintainer: Muammar El Khatib <muammar@debian.org>
+Build-Depends: debhelper (>= 9),
+               libcegui-mk2-dev (>= 0.6.2),
```

Comment:
TSC in `devel` now requires CEGUI >= 0.8.0, and beware of a fatal glm incompatibility explained at https://bitbucket.org/cegui/cegui/issues/1063/openglrenderer-on-linux-renders-cegui and https://github.com/Secretchronicles/TSC/issues/10#issuecomment-170358861. If Debian has glm >= 0.9.6, then CEGUI must be greater 0.8.5.